edb/lang/edgeql/parser/grammar/precedence.py

```diff
@@ -109,6 +109,22 @@ class P_TYPEAND(Precedence, assoc='left', tokens=('AMPER',)):
     pass
 
 
+class P_EXISTS(Precedence, assoc='right', tokens=('EXISTS',)):
```

Comment:
Please make these all equal (`rel_to_last='='`)

Reply:
OK, I've updated the precedence.